site/sass/components/_featured-content.scss

```diff
@@ -40,20 +74,17 @@
     }
   }
 
-  &__image-wrapper {
-    --circle-type: featured;
-    border-image-outset: 300px;
-    border-image-slice: 0 fill;
-    border-image-source: paint(circles);
+  &__image {
+    display: block;
+    height: 36vh;
+    max-height: 15rem;
+    min-height: 10vh;
+    object-fit: cover;
+    width: 100%;
 
-    img {
-      display: block;
-    }
-
-    > img:not(:first-of-type) {
-      left: 0;
-      position: absolute;
-      top: 0;
+    @include mq($bkpt-articles--2-cols) {
```

Comment:
If the height and min-height stuff from the wrapper is moved here and migrated to `clamp` we should be able to do away with this breakpoint entirely

Reply:
Yup, no longer there. Thanks!